evergreen.yaml

```diff
@@ -132,6 +166,23 @@ tasks:
   - <<: *run-build
     tags: ["test"]
     name: test-perf
+  - name: test-auth
+    commands:
+      - func: get-project
+      - func: set-up-mongodb-auth
+      - func: set-up-curator
+      - func: run-make
+        vars: { target: "cedar" }
+      - func: run-make
+        vars: { target: "clean-results" }
+      - func: run-make
+        vars: { target: "test-cedar", RUN_TEST: "TestGlobalEnvironmentAuth", TEST_USER_ADMIN: "myUserAdmin"}
```

Comment:
Can you format the same way you formatted the `vars` in `self-tests.yaml` in evg?

Reply:
Yeah good call